modules/mongodb/charts.go

```diff
@@ -432,3 +441,109 @@ var (
 		},
 	}
 )
+
+var (
+	chartDBStatsCollections = &module.Chart{
+		ID:    "database_collections",
+		Title: "Collections",
+		Units: "collections",
+		Fam:   "database_statistics",
+		Ctx:   "mongodb.database_collections",
+		Type:  module.Stacked,
+	}
+
+	chartDBStatsIndexes = &module.Chart{
+		ID:    "database_indexes",
+		Title: "Indexes",
+		Units: "indexes",
+		Fam:   "database_statistics",
+		Ctx:   "mongodb.database_indexes",
+		Type:  module.Stacked,
+	}
+
+	chartDBStatsViews = &module.Chart{
+		ID:    "database_views",
+		Title: "Views",
+		Units: "views",
+		Fam:   "database_statistics",
+		Ctx:   "mongodb.database_views",
+		Type:  module.Stacked,
+	}
+
+	chartDBStatsDocuments = &module.Chart{
+		ID:    "database_documents",
+		Title: "Documents",
+		Units: "documents",
+		Fam:   "database_statistics",
+		Ctx:   "mongodb.database_documents",
+		Type:  module.Stacked,
+	}
+
+	chartDBStatsSize = &module.Chart{
+		ID:    "database_storage_size",
+		Title: "Disk Size",
+		Units: "bytes",
+		Fam:   "database_statistics",
+		Ctx:   "mongodb.database_storage_size",
+		Type:  module.Stacked,
+	}
+)
+
+// updateDBStatsCharts adds dimensions for new databases and
+// removes for dropped
+func (m *Mongo) updateDBStatsCharts(databases []string) {
+	if len(databases) == 0 {
+		return
+	}
```

Comment:
I think we shouldn't early return here.

- 4 databases in total
- 4 => filtering => 1
- we add 1 db
- a user removes that db
- 3 databases in total
- 3 => filtering => 0
- we don't remove the db

Reply:
good point, fixed